cli/cmd/policy.go

```diff
@@ -124,6 +124,16 @@ To view the LQL query associated with the policy, use the query id shown.
 		Args:    cobra.ExactArgs(1),
 		RunE:    showPolicy,
 	}
+
+	// This is an experimental command.
+	// policyDisableTagCmd represents the policy disable-tags command
+	policyDisableTagCmd = &cobra.Command{
+		Use:     "disable-tags <tag>",
```

Comment:
So this was what I was envisioning however `disable-tag` sorta implies that your disabling a tag, as opposed to disabling policies based on a tag.

Could we do something like (and tag as a flag)?
`disable [policy_id]`

Meaning you can either disable a policy by ID, or a set of policies based on tag?

Reply:
👍  I like that idea

Reply:
I've refactored the cmd with the tag flag

lacework policy disable [policy_id]
lacework policy enable [policy_id]

lacework policy disable --tag [tag_name]
lacework policy enable --tag [tag_name]